adapterearthengine/services/query_service.py

```diff
@@ -19,6 +19,7 @@ def convert(query, query_type='sql'):
         endpoint = 'fs2SQL'
 
     result = endpoint+query
+    logging.info(result)
```

Comment:
add a small string before the actual value, so it's easy to identify, in the logs, that a given log line was produced by this `logging.info` call

Reply:
Changed as requested on the next commit: https://github.com/resource-watch/adapter-earth-engine/pull/12/commits/ef6191631dfaf8c6c50d1356d83dd2bd129b3462